docs/projects/version-control/config-as-code-reference.md

```diff
@@ -213,17 +214,22 @@ connectivity_policy {
 versioning_strategy {
 
     donor_package {
-        step = "Deploy a Package"
+        step = "deploy-a-package"
     }
 }
 ```
 
+## Slugs in OCL
+
+Previously, version controlled projects would reference shared resources using their name. This had a side-effect causing API responses for version controlled projects to contained names instead of IDs.
```

Comment:
```suggestion
Previously, version controlled projects would reference shared resources using their name. This had a side-effect causing API responses for version controlled projects to contain names instead of IDs.
```